web-admin/src/features/projects/status/ProjectResourcesTable.svelte

```diff
@@ -0,0 +1,116 @@
+<script lang="ts">
+  import * as Table from "@rilldata/web-common/components/table-shadcn";
+  import Tag from "@rilldata/web-common/components/tag/Tag.svelte";
+  import { prettyResourceKind } from "@rilldata/web-common/features/entity-management/resource-selectors";
+  import type { V1Resource } from "@rilldata/web-common/runtime-client";
+  import {
+    Render,
+    Subscribe,
+    createRender,
+    createTable,
+  } from "svelte-headless-table";
+  import { readable } from "svelte/store";
+  import ResourceErrorMessage from "./ResourceErrorMessage.svelte";
+  import {
+    getResourceKindTagColor,
+    prettyReconcileStatus,
+  } from "./display-utils";
+
+  export let resources: V1Resource[];
+
+  const table = createTable(readable(resources));
+
+  const columns = table.createColumns([
+    table.column({
+      accessor: (resource) => resource.meta.name.kind,
+      header: "Kind",
+      cell: ({ value }) => {
+        const prettyKind = prettyResourceKind(value);
+        const color = getResourceKindTagColor(value);
+        return createRender(Tag, {
+          color,
+        }).slot(prettyKind);
+      },
+    }),
+    table.column({
+      accessor: (resource) => resource.meta.name.name,
+      header: "Name",
+    }),
+    table.column({
+      accessor: (resource) => resource.meta.reconcileStatus,
+      header: "Execution status",
+      cell: ({ value }) => prettyReconcileStatus(value),
+    }),
+    table.column({
+      accessor: (resource) => resource.meta.reconcileError,
+      id: "error",
+      header: "Error",
+      cell: ({ value }) =>
+        createRender(ResourceErrorMessage, { message: value }),
+    }),
+    table.column({
+      accessor: (resource) => resource.meta.stateUpdatedOn,
+      header: "Last refresh",
+      cell: ({ value }) =>
+        new Date(value).toLocaleString(undefined, {
+          month: "short",
+          day: "numeric",
+          hour: "numeric",
+          minute: "numeric",
+        }),
+    }),
+    table.column({
+      accessor: (resource) => resource.meta.reconcileOn,
+      header: "Next refresh",
+      cell: ({ value }) => {
+        if (!value) {
+          return "-";
+        }
+        return new Date(value).toLocaleString(undefined, {
+          month: "short",
+          day: "numeric",
+          hour: "numeric",
+          minute: "numeric",
+        });
+      },
+    }),
+  ]);
+
+  const { headerRows, pageRows, tableAttrs, tableBodyAttrs } =
+    table.createViewModel(columns);
+</script>
+
+<div class="rounded-md border">
+  <Table.Root {...$tableAttrs}>
+    <Table.Header>
+      {#each $headerRows as headerRow}
+        <Subscribe rowAttrs={headerRow.attrs()}>
+          <Table.Row>
+            {#each headerRow.cells as cell (cell.id)}
+              <Subscribe attrs={cell.attrs()} let:attrs props={cell.props()}>
+                <Table.Head {...attrs}>
+                  <Render of={cell.render()} />
```

Comment:
This doesnt feel like svelte, more like react. Does the library not support slots?

Reply:
It doesn't seem to. All examples use `<Render />` and the [docs](https://svelte-headless-table.bryanmylee.com/docs/getting-started/quick-start#applying-to-markup) show this info box:
<img width="742" alt="image" src="https://github.com/rilldata/rill/assets/14206386/a618c615-e036-4244-b243-1020cf229a39">